src/pyhf/pdf.py

```diff
@@ -651,5 +748,16 @@ def logpdf(self, pars, data):
             raise
 
     def pdf(self, pars, data):
+        """
+        Compute the value of the full density.
```

Comment:
What is "value"? What is "full density"?

Reply:
the full model (constraint + main).. value is the density at a given observed point in data space

Reply:
Change the phrasing.

Reply:
to?

Reply:
"Compute the density at a given observed point in data space of the full model"